src/App.js

```diff
@@ -1,11 +1,15 @@
 import './App.css';
+import React from 'react';
+import Calculator from './calculator';
 
```

Comment:
Let's create the Display component and import it here...

![image](https://user-images.githubusercontent.com/31897434/130066377-19d2a7b2-4be3-4c64-b1e1-e1ab113592ca.png)